components/Volunteer/VolunteerDashboard.tsx

```diff
@@ -29,12 +30,38 @@ const VolunteerDashboard = ({
 }) => {
   // set pop up window to false
   const [showPopup, setShowPopup] = useState(false);
+  const [messageApi, contextHolder] = message.useMessage();
+
+  // Display success message
+  const successMessage = (message: string) => {
+    messageApi.open({
+      type: 'success',
+      content: message,
+    });
+  };
+
+  // Display error message
+  const errorMessage = (message: string) => {
+    messageApi.open({
+      type: 'error',
+      content: message,
+    });
+  };
```

Comment:
Use antd message [here](https://ant.design/components/message) to display success/error message in frontend.